tests/embedding_lookup.c

```diff
@@ -0,0 +1,29 @@
+#include <finalfusion.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <limits.h>
+#include <sys/param.h>
+
+int main() {
+  ff_embeddings embeds = ff_read_embeddings("test.fifu");
+  if (embeds == NULL) {
+    return 1;
+  }
+  uintptr_t dimensions = dims(embeds);
+
+  // in vocab
+  float buffer[dimensions];
```

Comment:
Alloc on the heap. People may copy such tests as examples, and this really blows up the stack. Also, this is invalid pre-C99 (the array size should be const).